packages/mgt-components/src/components/mgt-person/mgt-person.scss

```diff
@@ -222,18 +232,100 @@ mgt-person .person-root {
       .user-presence {
         left: calc(#{$avatar-size-s} - 8px);
         top: calc(#{$avatar-size-s} - 8px);
-        width: 8px;
-        height: 8px;
+        width: 6px;
+        height: 6px;
 
         .presence-basic {
-          width: 4px;
-          height: 4px;
-          border-radius: 4px;
+          width: 2px;
+          height: 2px;
+          border-radius: 50%;
           padding: 0;
+          font-size: calc(#{$avatar-size-s} * 0.14);
 
           & .presence-oof-offline svg {
-            height: 0;
-            width: 0;
+            width: calc(#{$avatar-size-s} * 0.14);
+            margin-left: calc(#{$avatar-size-s} * 0.07 - 2px);
+          }
+        }
+
+        .presence-offline::before,
+        .presence-available::before,
+        .presence-oof-available::before,
+        .presence-away::before,
+        .presence-oof-away::before,
+        .presence-dnd::before,
+        .presence-oof-dnd::before,
+        .presence-oof-offline::before {
+          margin-left: calc(#{$avatar-size-s} * 0.07 - 2px);
```

Comment:
This offline style for the presence is a little off:
![image](https://user-images.githubusercontent.com/7122716/179569436-53e73712-cd4e-4354-a488-28c53e33c820.png)
(Comment also added to #1619 )